radar-upload-frontend/src/components/Login/index.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <v-row
+    align="center"
+    justify="center"
+  >
+    <v-col
+      cols="12"
+      sm="8"
+      md="4"
+    >
+      <v-card class="elevation-12">
+        <v-toolbar
+          color="primary"
+          dark
+          flat
+        >
+          <v-toolbar-title>Login form</v-toolbar-title>
+        </v-toolbar>
+        <v-card-text>
+          <v-form :disabled="loading">
+            <v-text-field
+              label="Login"
```

Comment:
I am a bit confused why do we have username and password here, when you will redirect to Managementportal to login?

Reply:
I just put it there so that the form is not too simple. And also in case we split into a separate app.

Reply:
I see. Though it is kind of leading to the user, since it doesn't do anything.

Reply:
Yes, I agree with Nivethika. Please redirect to managementportal, if we intend to change the login behaviour we can always change it back to this at a later time.